src/labels/CustomLabels.labels

```diff
@@ -5867,6 +5995,14 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
         <shortDescription>stgLabelMembershipSettings</shortDescription>
         <value>Membership Settings</value>
     </labels>
+    <labels>
+        <fullName>stgLabelName</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>stgLabelName</shortDescription>
+        <value>Name</value>
```

Comment:
True, I think we put this in since the DeveloperName on all of the objects didn't say just 'Name' (like Custom Metadata Record Name). This is being appended to the rollup and filter group labels for the 'Name' display on both of these. Should we change the CMT record instead?

Reply:
Not clear on usage though. Is this the label for a Rollup or something else in Settings. The custom label name is `stgLabelName` which just says it's a name, but it could be any name on any object. I think what I'm noting is that the Custom Label name doesn't describe what it's intended to represent. Someone could use this label for something else that requires "Name" (such as Contact Name), but the the intent (and possibly the translation) might not match that usage.

Reply:
So one place we're using it is on the Detail view for a rollup, and generating the label and the name for the name input with: label="{!v.labels.rollupLabel + ' ' + v.labels.name}". Since we're binding the with: value="{!v.activeRollup.MasterLabel}", we probably should be using the field label for MasterLabel, I just don't know that calling it a 'Label' for the customer makes sense. We can discuss in today's meeting too! Any thoughts on this @bethbrains?

Reply:
Ah. I see. So, the Rollup Label is the auto-generated/user-entered "name" for the rollup, correct? If so, then MasterLabel might be something like "Sum Amount to Total Donations" or "Last CloseDate to Last Donation Date". If that's the case it might seem odd to show that on the page as "Last CloseDate to Last Donation Date Name". Unless I'm mis-understanding the usage. Maybe this is for the field label, so it might generate something like "Label Name"?

Reply:
So this would just be for the field label, so it would show "Rollup Name" above the field that would then have the generated "GAU: First Donation" value.